content/threads/weekly-thread-2021-01.md

```diff
@@ -1,11 +1,11 @@
 ---
 slug: weekly-thread-2021-01
```

Comment:
Can you cmd shift f replace `weekly-thread-2021-01` with `weekly-thread-2022-01` in all files, and then rename this one?

Reply:
Yeah